tools/python_tests/permissions_test.py

```diff
@@ -27,11 +27,18 @@
     "uses-permission: com.amazon.device.messaging.permission.RECEIVE",
 }
 
+SPLIT_RE = re.compile("[\.\-]")
+APK_RE = re.compile(".*universal(?!.*?unaligned).*$")
+PROP_RE = re.compile("(?!\s*?#).*$")
+CLEAN_PERM_RE = re.compile("(name='|'$)", re.MULTILINE)
+
+
 logger = logging.getLogger()
 logger.level = logging.DEBUG
 logger.addHandler(logging.StreamHandler(sys.stdout))
 
-def new_lines(iterable):
+
+def join_by_new_lines(iterable):
```

Comment:
join_with_new_lines?
And you can make it a local function, cause you use it only in one place.